doc/swss-schema.md

```diff
@@ -101,10 +101,13 @@ For example (reorder output)
 ###VLAN_TABLE
     ;Defines VLANs and the interfaces which are members of the vlan
     ;Status: work in progress
-    key                 = DEVICE_NAME    ; must be unique across PORT,INTF,VLAN,LAG TABLES
-    vland_id            = DIGIT 0-4095
-    admin_status        = BIT ; administrative status for vlan up or down
-    attach_to           = PORT_TABLE.key
+    key                 = VLAN_TABLE:vlanid	; DIGIT 0-4095
+    admin_status        = "down" / "up"		; admin status
+    oper_status		= "down" / "up"		; operating status
+    
+    key			= VLAN_TABLE:vlanid:membername	; vlan member
+    oper_status		= "down" / "up"		; operating status
```

Comment:
why oper status is needed here? what's the difference between this one and the physical port oper status?